### DISCUSSION TITLE
Time of last touch on the screen

### DISCUSSION THREAD

**Discussion by AK1Chel** [arendst/Tasmota#21355] [General]
Please tell me how to determine the time of the last touch on the screen in Berry

**Comment by AK1Chel** [comment]
I want to find an analogue of Short Idle and Long Idle like Haspmota

**Comment by s-hadinger** [comment]
If you want to measure the time difference between touches on buttons, you need to react to button events.

If you want just the last time there was an interaction, for ex for a screen saver, see : https://tasmota.github.io/docs/LVGL/#measuring-user-inactivity

`lv.disp().get_inactive_time()`

**Reply by AK1Chel** [reply]
Thanks for the help
